landlab/grid/hex.py

```diff
@@ -235,24 +246,33 @@ def _initialize(
                 + ")"
             )
 
+        # Pre-LL2.0: handle shape and node_layout keyword args
+        # (note: in LL2.0, shape should refer ONLY to rows and columns)
+        if type(shape) is str:  # "old" (LL < 2.0) usage
+            node_layout = shape
+            warn(message="Use node_layout to specify 'rect' or 'hex' layout",
+                 category=DeprecationWarning)
+        elif type(shape) is tuple:
+            (base_num_rows, base_num_cols) = shape
+
```

Comment:
@gregtucker my only comments are to:

- [ ] Add a statement to the deprecation warning that indicates that this change will be enforced in LL2.0, and that shape will have a new meaning in LL2.0
- [ ] Add a test that the deprecation warning is raised correctly.
- [ ] Add a test that if shape is passed a tuple, it behaves as expected. 

I couldn't find tests in the PR, and I double checked that [coveralls indicates](https://coveralls.io/builds/22923077/source?filename=landlab/grid/hex.py) that those lines are still untested.

Reply:
Good ideas, thanks---now added.